Automattic/jetpack
# Jetpack AI Voice to content: Update to modal UI

Part of #35547

## Proposed changes:

* Updates the UI for a modal, according to Figma specs

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Go to the block editor
* Add a Voice to content block
* Check the UI
